internal/apigw/apigw_openapi.go

```diff
@@ -171,6 +175,28 @@ func (module *Module) buildOperation(ctx pgsgo.Context, method pgs.Method, mt *m
 	return r, op, components, nil
 }
 
+func getTerraformEntityOperationExtension(operation *apigw_v1.Operation) string {
+	terraformEntity := ""
+	if operation.TerraformEntity == nil {
+		return ""
+	}
+	switch operation.TerraformEntity.Type {
```

Comment:
Are we missing `GET` in this switch statement? Does the default case handle that?

Reply:
Clarified with the speakeasy team and `get` is an alias for `read`, so I removed `GET` in the latest commit.